### PR TITLE
Feature/add task

### PR DESCRIPTION
**Pull request by Edona94**

### CODE REVIEW

File: backend/src/main/java/de/neuefische/timemanagement/backend/service/TaskService.java [hunk 1]
```diff
@@ -5,15 +5,29 @@
...
     private final TaskRepo taskRepo;
+    private final IdService idService;
 
     public List<Task> getAllTasks(){
         return taskRepo.getAllTasks();
     }
+
+    public Task addTask(Task newTask){
+        if(newTask.title()==null ||newTask.title().equals("") ||newTask.dateTime()==null){
+            throw new IllegalArgumentException("missing title or date");
```
Comment (HendrikNeuschulz) [inline backend/src/main/java/de/neuefische/timemanagement/backend/service/TaskService.java:25]: maybe you should handle this Exception with a catch and response Information about the Exception to your frontend.

File: backend/src/test/java/de/neuefische/timemanagement/backend/service/TaskServiceTest.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.Mock;
```
Comment (HendrikNeuschulz) [inline backend/src/test/java/de/neuefische/timemanagement/backend/service/TaskServiceTest.java:9]: empty line ...

**Review by HendrikNeuschulz — APPROVED** [review]
Nice work ! Only a few comments. Please have a look - we trust you will implement the suggestions.

**Comment by Anmavel** [comment]
Thank you for your Feedback :-)

**Pull request merged**